scripts/actions/__tests__/kitchen-sink.mdx

```diff
@@ -32,7 +32,7 @@ For an index of documentation available during the private Beta, see the [C SDK
 To instrument errors in transactions:
 
 1. [Start a transaction](/docs/instrument-transactions-c-agent#start-transaction).
-2. Record an error with [`newrelic_notice_error()`](https://source.datanerd.us/c-agent/c-agent/blob/master/examples/ex_notice_error.c), supplying the required parameters.
+2. Record an with [`newrelic_notice_error()`](https://source.datanerd.us/c-agent/c-agent/blob/master/examples/ex_notice_error.c), supplying the required parameters.
```

Comment:
this was removed during a test on `mark` and `var`! if it's really important I can add it back in but the snapshot has already been updated :)